trashcli/list.py

```diff
@@ -12,16 +15,45 @@
 from .trash import unknown_date
 
 def main():
-    import sys
-    import os
     from trashcli.list_mount_points import os_mount_points
     ListCmd(
         out          = sys.stdout,
         err          = sys.stderr,
         environ      = os.environ,
         getuid       = os.getuid,
         list_volumes = os_mount_points,
-    ).run(*sys.argv)
+    ).run(sys.argv)
+
+def parse_args(sys_argv, curdir):
+    import argparse
+    parser = argparse.ArgumentParser(
+        description='List trashed files',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--sort',
+                        choices=['date', 'path', 'none'],
+                        default='date',
+                        help='Sort list of list candidates by given field')
+    parser.add_argument('--trash-dir',
+                        action='store',
+                        dest='trash_dir',
+                        help=argparse.SUPPRESS)
```

Comment:
Please don't add command options that are not used.

Reply:
Yep